Add information on querying Ref Tables in Logs (#18829)
DataDog/documentation: content/en/logs/explorer/analytics/_index.md

To apply formulas with multiple queries, all queries must be grouped by the same facet. In the example above, both queries are grouped by `Webstore Store Name`.

You can apply a function to a formula by clicking on the `Σ` icon. Here is an example of how to apply a [Timeshift function][12] on the proportion of error logs in all logs to compare current data with data from one week before:

{{< img src="logs/explorer/group/timeshift_function_logs.jpg" alt="The query editor showing a formula with the week before timeshift function applied to it" style="width:100%;" >}}

## Filter logs based on Reference Tables

> <div class="alert alert-warning">Reference Tables and the feature to filter logs based on Reference Tables are in beta. There are no billing implications for defining and querying Reference Tables. There is a limit of 100 Reference Tables per account and a limit of 1000 rows in a Reference Table.</div>

Reviewer: There may be costs to defining a reference table in the future so let's leave that part out.

Author: ```suggestion
<div class="alert alert-warning">Reference Tables and the feature to filter logs based on Reference Tables are in beta. There is a limit of 100 Reference Tables per account and a limit of 1000 rows in a Reference Table.</div>
```

Reference Tables allow you to combine metadata with logs, providing more information to resolve application issues. Add a query filter based on a Reference Table to perform lookup queries. For more information on creating and managing this feature, see the [Reference Tables][17] guide.

> To apply a query filter with Reference Tables, click on the `+ Add` button next to the query editor and select **Reference Table**. In the following example, the Reference Table query filter is used to search all recent logs that include a malicious IP address from a thread intel reference table:

Reviewer 2: ```suggestion
To apply a query filter with Reference Tables, click on the `+ Add` button next to the query editor and select **Reference Table**. In the following example, the Reference Table query filter is used to search all recent logs that include a malicious IP address from a threat intel reference table:
```

> {{< img src="/logs/explorer/group/reference_tables_logs.png" alt="Use Reference Tables in the Log Explorer" border="true" popup="true" style="width:100%;" >}}

Reviewer 2: ```suggestion
{{< img src="/logs/explorer/group/reference_tables_logs.png" alt="The Datadog log explorer with reference table search options highlighted" border="true" popup="true" style="width:100%;" >}}
```
